### ISSUE TITLE
Script to Load the Data into local and EC2 database

### ISSUE THREAD

**Issue by cl0ckwork** [cl0ckwork/dw-timescale-db#2]

**Comment by cl0ckwork** [comment]
@iamdeepaka we need to load the data into the remote database, the `docker-compose` file already uses a timescale image, so we to load the remote database already running the image.

i like the use of the `.sql` file, ideally there is a way to load the data remotely. otherwise we can ssh in and do it manually.

there is already a database name as well, you can see the configuration in this file `/lib/db/py`

```python
    dbname='data-warehousing',
    user='postgres',
    password='password',
    host='ec2-34-207-126-253.compute-1.amazonaws.com',
    port='5432'
```

**Comment by deepakag5** [comment]
@cl0ckwork I am trying to connect the database with given connection string , username and password on MySQL Workbench but unable to connect, you can see below error. Let me know how can I connect the database so that I can load the data.

![dberror](https://user-images.githubusercontent.com/32446623/48677480-9df1b600-eb43-11e8-8c84-73e98500ac33.png)

Meanwhile, I have updated the tweets.sql file as per the cleaned data if you can connect to database you can use it to load the data. Thanks

**Comment by cl0ckwork** [comment]
@iamdeepaka are you on the VPN when trying to connect?

Otherwise, you can check the EC2 instances to make sure the instance is running, it was running yesterday, but free tier isn't that reliable.

**Comment by deepakag5** [comment]
@cl0ckwork no I am trying to connect from my home. I can't have a VPN as college doesn't support VPN for linux system. Can you use the sql file I created to load the data. I will write queries for this data on my local system then we can run on EC2. Let me know.

**Comment by cl0ckwork** [comment]
@iamdeepaka I dont believe you can connect to GWU's AWS without the VPN. I was able to load the data last night.

if you want, you can try this walk through:
https://faq.oit.gatech.edu/content/how-do-i-install-cisco-anyconnect-client-linux

this is our vpn address: go.vpn.gwu.edu

**Comment by cl0ckwork** [comment]
closing since remote server has data.

**Comment by deepakag5** [comment]
Thanks for sharing the link, but as I mentioned earlier our College GWU doesn't support linux. I have mailed long time back to GW IT department and they confirmed I cannot use ubuntu to connect the VPN. So, I am not going to put time into it. 

Also, I have created bash script which automatically install timescaledb into docker locally, create database and tables, load data into it. Now doing it on EC2 is just using basic networking concepts which I already know. If I wish to do that I have my own AWS account to play around. 

Hope this helps.
